### ISSUE TITLE
isProtocolUpToDate error

### ISSUE THREAD

**Issue by azazellochg** [I2PC/scipion#1100]
In 1.1.1 branch for some reason femtoseconds are not written in project.sqlite for some of lastUpdateTimeStamp values of the protocols... can't figure out why
```
Exception in Tkinter callback
Traceback (most recent call last):
  File "/lmb/home/gsharov/soft/scipion/software/lib/python2.7/lib-tk/Tkinter.py", line 1486, in __call__
    return self.func(*args)
  File "/lmb/home/gsharov/soft/scipion/software/lib/python2.7/lib-tk/Tkinter.py", line 533, in callit
    func(*args)
  File "/lmb/home/gsharov/soft/scipion/pyworkflow/gui/project/viewprotocols.py", line 806, in _automaticRefreshRuns
    if self.project.needRefresh():
  File "/lmb/home/gsharov/soft/scipion/pyworkflow/project.py", line 957, in needRefresh
    if not pwprot.isProtocolUpToDate(run):
  File "/lmb/home/gsharov/soft/scipion/pyworkflow/protocol/protocol.py", line 1797, in isProtocolUpToDate
    protTS = protocol.lastUpdateTimeStamp.datetime()
  File "/lmb/home/gsharov/soft/scipion/pyworkflow/object.py", line 664, in datetime
    return dt.datetime.strptime(self._objValue, formatStr)
  File "/lmb/home/gsharov/soft/scipion/software/lib/python2.7/_strptime.py", line 325, in _strptime
    (data_string, format))
ValueError: time data '2017-07-05 17:52:35' does not match format '%Y-%m-%d %H:%M:%S.%f'
```

**Comment by azazellochg** [comment]
I think this was (possibly) introduced in 6c919c5

**Comment by pconesa** [comment]
Thanks Gregory, it rings a bell. Roberto I think showed me that, and not 
sure if it's already fixed. I'll check.

This isProtocolUpToDate is appearing to many time recently (myfault). 
I'll refactor it to be safer.

**Comment by pconesa** [comment]
So this is the bugfix!

**Comment by azazellochg** [comment]
when I commented out this commit it the problem is gone.

**Comment by azazellochg** [comment]
Fixed in #1139
